test/network.spec.js

```diff
@@ -189,15 +190,15 @@ describe('network', function() {
       const { page, server } = getTestState();
 
       const response = await page.goto(server.PREFIX + '/simple.json');
-      expect(await response.text()).toBe('{"foo": "bar"}\n');
+      expect(await response.text()).toBe(`{"foo": "bar"}${os.EOL}`);
```

Comment:
Why do we need the `os.EOL`?

The `simple.json` file is checked in to our repository and uses LF for line endings. Our `.gitattributes` enforces LF line endings on checkout as well. How could we still end up with CRLF in server responses?

Reply:
Hmm I'm not sure - I presumed this was maybe due to OS level differences when the server fetches the JSON file but I'm not sure if that makes sense or not thinking about it.

Reply:
Without this change, does the test fail?

Reply:
Yes it does.  We'll `trimEnd()` it to avoid any line ending faff 👍